doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -0,0 +1,240 @@
+---
+title: operator-testing-tool
+authors:
+  - "@jmccormick2001"
+  - "@joelanford"
+reviewers:
+  - TBD
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-13
+last-updated: 2019-09-13
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# operator-testing-tool
+
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created.
+
+## Summary
+
+This proposal is for a new Operator Testing Tool that validates Operator packaging as well as run simple functional test specific to the Operator’s APIs.
+
+## Motivation
+
+Operator Testing and Validation remains difficult today due to tooling sprawl, disparate definitions of validity, and the lack of means to provide custom tests per Operator in a way that is not tied to a particular CI pipeline.
+
+Having high quality Operators is crucial to the success of the Operator Framework.
+
+### Goals
+
+ * Operator Developers can use a tool that reports on recommended, required and optional packaging requirements of a given bundle on disk
+ * Operator Developers can use the same tool that reports on recommended, required and optional common black box tests of a deployed operator on cluster
+ * Operator Developers can rely on a central source of truth of a valid Operator bundles shared among all Operator Framework components
+
+
+### Non-Goals
+
+not in scope: Operator Developers can use the same tool to run custom, functional tests of their Operator on cluster
+
+## Proposal
+
+### User Stories 
+
+#### Story 1 - Show pass/fail in Scorecard Output
```

Comment:
For all user story, IMHO we need to update them in order to follow its common defintiions as https://www.berteig.com/how-to-apply-agile/summary-of-user-stories-the-three-cs-and-invest/

Reply:
@dmesser has been pushing this point as well. I don't think we should require this method of writing stories in the proposals this time around because it wasn't discussed as a team, and there are still some kinks to work out with how these things are translated into JIRA issues.

Let's talk about this as a team for the 4.4+ epics and planning.